### ISSUE TITLE
Does weight decay apply on distillation token?

### ISSUE THREAD

**Issue by chunfuchen** [facebookresearch/deit#60]
As title. 

I wonder that do you add L2 weight decay on the distillation token?
The vision transformer excludes cls_token and pos_embed in computing weight decay. 
https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/vision_transformer.py#L347

Thanks.

**Comment by TouvronHugo** [comment]
Hi @chunfuchen,
Yes we add weight decay on the distillation token (see [issue 48](https://github.com/facebookresearch/deit/issues/48)).
Best,
Hugo
